Sources/StreamChatUI/MessageActionsPopup/ChatMessagePopupVC_Tests.swift

```diff
@@ -7,7 +7,11 @@ import StreamChatTestTools
 @testable import StreamChatUI
 import XCTest
 
+#warning("Is this OK for the test purpose?")
+extension _ChatMessagePopupVC: AppearanceProvider {}
```

Comment:
The `ChatMessagePopupVC` itself doesn't need the appearance, but for the testing, I added this extension, is that ok or it's better to conform to the `AppearanceProvider` directly in that view?